codecov/settings_prod.py

```diff
@@ -1,11 +1,5 @@
 import os
 
-import sentry_sdk
```

Comment:
Was this never used or?

Reply:
All these imports already exist in `settings_base.py`.
I assumed they were reduntant because settings_base is imported in `settings_prod`